site/docs/v1/tech/identity-providers/samlv2-idp-initiated/okta.adoc

```diff
@@ -126,13 +126,60 @@ Return to the Configure SAML Tab.  This can be found by navigating as follows fr
 
 * Set the value of the [field]#Single sign on URL# to the value of the [field]#Callback URL (ACS)# from the FusionAuth Identity Provider recorded above.
 * Set the value of the [field]#Audience URI (SP Entity ID)# to the value of the [field]#Issuer# recorded above.
-* Optionally set the value of [field]#Default RelayState# if you want to provide a specific redirect URI. If this value is omitted, the first Authorized redirect URI found in the FusionAuth Application OAuth configuration will be used. This option is supported in FusionAuth version greater than or equal to `1.41.0`. In prior versions, append `redirect_uri=` to the value specified [field]#Single sign on URL# to control the final redirect URI.
+* Optionally set the value of [field]#Default RelayState# if you want to provide a specific redirect URI. If this value is omitted, the first Authorized redirect URI found in the FusionAuth Application OAuth configuration will be used. This option is supported in FusionAuth version greater than or equal to `1.41.0`. In prior versions, append `redirect_uri=` to the value specified [field]#Single sign on URL# to control the final redirect URI.  For more information link:/docs/v1/tech/identity-providers/samlv2-idp-initiated/okta#optional-adding-a-specific-redirect-after-saml-login[see section below].
 * Set the [field]#Application username# to be `Email`.
 
 image::identity-providers/idp-initiated/okta-sign-on-advanced-settings.png[Configure Okta with the FusionAuth SP information,width=1200]
+There are a few optional workflows (adding a persistent Id, sending over an email attribute, and configuring a custom redirect) you can employ listed below in the next *two* sections.  If these are not needed, please save the Okta application by clicking [breadcrumb]#"Next"# until complete and skip down.
 
-Save the Okta application by clicking [breadcrumb]#"Next"# until complete.
+=== Optional: Adding a Specific Redirect After SAML login
 
+*Scenario:* you want a user to return to a specific redirect url after SAML auth is complete, there are two options to specify this url:
+
+1. Add to Relay State field
+- As of version `1.41.0` FusionAuth allows adding a redirect in the relay state.
+- Simply add something like `https://hooli.com` to the [field]#Default RelayState# in your Okta configuration.
+2. Add to the ACS
+- You can append your redirect to the [field]#Single sign-on URL# (ACS)
+- Example: +
+`<host>/samlv2/acs/22cea679-83af-422a-806a-baae792b3ab9/85a03867-dccf-4882-adde-1a79aeec50df?redirect_uri=http://hooli.com`
```

Comment:
Technically `http://hooli.com` would need to be URL encoded here which is `http%3A%2F%2Fhooli.com`.